.github/workflows/todo.yml

```diff
@@ -0,0 +1,21 @@
+name: To Do app CI & CD
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    env:
+      TOKEN: ${{ secrets.trelloToken}}
+      KEY: ${{ secrets.trelloKey}}
```

Comment:
(as per below, you'll also need to set that SECRET_KEY variable here)

Reply:
SECRET_KEY has been added and UI test is passing now